scripts/src/main/resources/scripts/command/docker

```diff
@@ -62,9 +62,11 @@ function doSetup() {
           doEnsureVpnKitInstallation
         elif doIsMacOs
         then
-          doRunCommand "cp -a '${rancher_install_dir}'/*.app /Applications/"
+          # shellcheck disable=SC2016
+          doRunCommand 'cp -a "${rancher_install_dir}"/*.app /Applications/'
```

Comment:
While this change seems sane it is not releated to Linux as indicated by the bug issue or the PR title.
Fine, but in such cases you may also leave a comment in the PR summary why this had to be changed.

Reply:
I linked the bug issue and indeed there it is described both in the title and in the issue description.
Hence, I am resolving this conversation.